# **Neural Networks implemented in Python's Tensor flow**

## **Part 3, our First Neural Networks:**

### **Section 1, :**

    Lets import our stuff again

In [8]:
import pandas as pd
import numpy as np
import seaborn as sns
import matplotlib.pyplot as plt

import tensorflow as tf
from tensorflow import keras
from tensorflow.keras import layers # type: ignore
import tensorflow.compat.v2.feature_column as fc # type: ignore
import tensorflow.compat.v1 as tf1 # type: ignore

    Our dataset for today is the fassion dataset from keras, imported as follows

In [12]:
fashion_mnist = keras.datasets.fashion_mnist
(train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()  # split test & train
train_images.shape

(60000, 28, 28)

    as we can see our dataset appears to be 60,000 images of 28*28 pixels
    so lets snoop 1 of these pixels

In [13]:
train_images[0,23,23]

194

    hmm, 194? that doesn't mean much, lets try this: